packages/fxa-auth-server/lib/routes/subscriptions/mozilla.ts

```diff
@@ -117,4 +132,28 @@ export class MozillaSubscriptionHandler {
       subscriptions: [...response.subscriptions, ...iapGooglePlaySubscriptions],
     };
   }
+
+  async getSubscriptionsForSupportPanel(request: AuthRequest) {
+    this.log.begin('mozillaSubscriptions.supportPanelSubscriptions', request);
+    const { uid } = request.query as Record<string, string>;
+    const iapPlaySubscriptions = await this.getPlaySubscriptions(uid);
+    const customer = await this.stripeHelper.fetchCustomer(uid);
```

Comment:
The `/account` route uses `this.stripeHelper.customer`. Does it make a difference? It looks like `customer` checks the Redis cache first, otherwise it also calls `fetchCustomer`.

Reply:
Ben has a patch that will remove all the customer Redis caching.  No difference, just some extra steps.